code/game/objects/structures/dresser.dm

```diff
@@ -5,6 +5,8 @@
 	icon_state = "dresser"
 	density = 1
 	anchored = 1
+	burn_state = FLAMMABLE
+	burntime = 25
```

Comment:
@FalseIncarnate I've made them deconstructable via unwrenching them, and crowbarring them with a long timer.